docs/tools/offline/cli-setup.md

```diff
@@ -31,37 +33,11 @@ Mbed CLI is compatible with [Virtual Python Environment (virtualenv)](https://py
 
 You may want to install Mbed CLI on a virtual environment if your main envrionment has an unsupported Python version.
 
-### Installing the stable version
-
-You can get the latest stable version of Mbed CLI through pip by running:
-
-```
-$ pip install mbed-cli
-```
-
-On Linux or Mac, you may need to run with `sudo`.
-
-#### Optional: installing the development version
-
-If you are interested in working with the development version (and perhaps contributing to Mbed CLI), clone the development repository [https://github.com/ARMmbed/mbed-cli](https://github.com/ARMmbed/mbed-cli):
-
-```
-$ git clone https://github.com/ARMmbed/mbed-cli
-```
-
-Once cloned, you can install Mbed CLI as a Python package:
-
-```
-$ python setup.py install
-```
-
-On Linux or Mac, you may need to run with `sudo`.
-
-#### Mbed CLI for Windows installer
+### Windows installer for Mbed CLI
 
-Mbed CLI for Windows installs [Mbed CLI](https://github.com/ARMmbed/mbed-cli) with all requirements on Windows 7 and newer (both the 32- and 64-bit versions).
+[Mbed CLI](https://github.com/ARMmbed/mbed-cli) is also bundled with a Windows installer that includes all requirements for Windows 7 and newer (both the 32- and 64-bit versions).
```

Comment:
Query: Who or what bundled Mbed CLI with the Windows installer? Please clarify for active voice.

Reply:
The Windows installer includes Mbed CLI (and other things).
This may require a bit more discussion in terms of the name for the product.
Maybe use [Mbed CLI Windows Installer](https://github.com/ARMmbed/mbed-cli-windows-installer)
FYI @arekzaluski